embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/EmbraceImpl.java

```diff
@@ -1300,21 +1300,6 @@ boolean endView(@NonNull String name) {
         return false;
     }
 
-    /**
-     * Logs the fact that a particular view was entered.
-     * <p>
-     * If the previously logged view has the same name, a duplicate view breadcrumb will not be
-     * logged.
-     *
-     * @param screen the name of the view to log
-     */
-    void logView(String screen) {
```

Comment:
Is this unused by RN & other callers?

Reply:
The only caller was the `logRnView` method so I've just inlined in. We still call the underlying method in `BreadcrumbService` internally, but the RN use case is the only public API we support to directly invoke this.